src/playout/mod.rs

```diff
@@ -74,21 +75,23 @@ pub trait Playout: Sync + Send {
         size as usize * size as usize * 3
     }
 
-    fn select_move(&self, board: &Board) -> Move {
+    fn select_move(&self, board: &Board, rng: &mut XorShiftRng) -> Move {
         let color = board.next_player();
         let vacant = board.vacant();
-        let playable_move_exists =  vacant
+        let playable_move =  vacant
```

Comment:
OK. Once this part of the code base has stabilised I'll investigate how to test it. Complicated stuff like this makes it a bit harder to test, of course. Once I remove the simple playouts and the configuration for adding the pass move it should already be considerably easier.